<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkInterface</class>
 <widget class="QWidget" name="NetworkInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnminimumwidth="120,100,100,0,0">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="1" column="2">
    <widget class="QSpinBox" name="uploadLimitCustom">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> KB/s</string>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="downloadLimitCmb">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="setLimitsBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="networkInterfaceLabel">
     <property name="text">
      <string>eth0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="downloadLimitCustom">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> KB/s</string>
     </property>
     <property name="maximum">
      <number>100000000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="uploadLimitCmb">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
